### ISSUE TITLE
Beat-Bildschirm erstellen

### ISSUE THREAD

**Issue by dspeckmann** [SWEProjekt/Freestyler-App#23]
Ein Bildschirm, der zwei Listen in Tabs enthält, die durch horizontales Wischen erreicht werden können. Die Linke zeigt alle lokal aufgenommenen Raps an und ermöglicht das Abspielen sowie Hochladen, die Rechte zeigt Beats anderer Nutzer aus der Datenbank an und ermöglicht zusätzlich das Bewerten dieser. Beide lassen sich nach verschiedenen Kriterien sortieren und filtern (Verwendeter Beat, verwendete Wörter, Bewertung).

**Comment by ScarlettMueller** [comment]
Wäre es hier nicht sinnvoller das ganze auf zwei Bildschirme aufzuteilen? So wie ich das verstanden habe, was du beschrieben hast, wird auch das sehr schnell unübersichtlich. Außerdem finde ich das beides auch thematisch besser getrennt wäre, bei zwei eignen Bildschirmen, da es ja doch Unterschiede gibt.

**Comment by dspeckmann** [comment]
Naja, abgesehen davon, dass die einen Raps von dir kommen und die anderen nicht, sehe ich da nicht so den großen Unterschied? Ich hätte mir das jetzt so vorgestellt, wie wenn du z.B. in WhatsApp zwischen Anrufen, Chats und Kontakten wechselst.

**Comment by ScarlettMueller** [comment]
Ok. Dann habe ich es einfach falsch verstanden. Für mich hat sich die Beschreibung so angehört, dass die Spalten nebeneinander sein sollen und nicht mit Tabs getrennt.

**Comment by dspeckmann** [comment]
Ne, da würde ich dir zustimmen, das wäre unübersichtlich!

**Comment by ScarlettMueller** [comment]
Brauchen wir noch zwei Tabs? Also können die Nutzer noch die Raps anderer Nutzer sehen?

**Comment by dspeckmann** [comment]
Nein, da die Online-Funktionalität noch nicht implementiert wird.
